railties/lib/rails/application/configuration.rb

```diff
@@ -283,6 +283,10 @@ def load_defaults(target_version)
           if respond_to?(:action_controller)
             action_controller.allow_deprecated_parameters_hash_equality = false
           end
+
+          if respond_to?(:active_record)
+            active_record.use_foreign_keys = true
+          end
```

Comment:
Do we need this given `true` is the default?

Reply:
We actually do, otherwise the config value would be `nil` when referenced like this:

```
irb(main):001:0> Rails.application.config.active_record.use_foreign_keys
=> nil
```